feature-libs/checkout/base/root/events/checkout.events.ts

```diff
@@ -105,31 +105,31 @@ export abstract class CheckoutDeliveryModeEvent extends CheckoutEvent {}
 /**
  * Emit this event to force delivery modes reload
  */
-export class ReloadDeliveryModesEvent extends CheckoutDeliveryModeEvent {
+export class CheckoutReloadDeliveryModesEvent extends CheckoutDeliveryModeEvent {
```

Comment:
Can we add a `TODO:#checkout - not used?` for `CheckoutReloadDeliveryModesEvent` and `ReloadCheckoutQueryEvent`? These don't seem to be dispatched anywhere

Reply:
I didn't add the todo because I was thinking that we set this up in the service in a way that a customer can always fire those events to 'reload' them anyhow if they ever need it

Reply:
I guess that was the idea. Not sure if it's realistic to actually be used. Maybe we can document it, so the events will have some exposure and a potential to be used.